### PR TITLE
User can belong to multiple instances

### PR DESCRIPTION
**Pull request by Neloop**

### CODE REVIEW

File: app/V1Module/security/Policies/GroupPermissionPolicy.php [hunk 1]
```diff
@@ -51,12 +52,17 @@ public function canAccessDetail(Identity $identity, Group $group): bool {
...
-        || ($user->getInstance() !== null
-            && $user->getInstance()->getRootGroup() !== null
-            && $group->getId() === $user->getInstance()->getRootGroup()->getId());
+    if ($group->isAdminOf($user)
+      || $group->isMemberOf($user)
+      || $group->isPublic()) {
+      return true;
+    }
+
+    return $user->getInstances()->exists(
```
[the remark this answers is not in the document]
Comment (Teyras) [reply app/V1Module/security/Policies/GroupPermissionPolicy.php]: Yes, but `isAdminOf` is okay :) What bothers me is `canAccessDetail`.

[the remark this answers is not in the document]
Comment (Neloop) [reply app/V1Module/security/Policies/GroupPermissionPolicy.php]: I figured... I thought the `$group->isAdminOf`, `$group->isMemberOf`, ... calls

**Comment by Teyras** [comment]
Are we still waiting for something before we merge this?

**Comment by Neloop** [comment]
Yes... implementation in web-app. This maybe will not be included in next release, so we really do not want to break things.

**Pull request merged**